src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -222,4 +222,8 @@ public static final class Config {
     public static final boolean MAPPING_INACTIVE = false;
     public static final String ISO = "ISO";
     public static final String FIPS = "FIPS";
+    
+    public class CuPaymentSourceConstants{
```

Comment:
Missing space between classname and curly brace.

Also, since this is a nested constants class, it should be declared as `static` and `final`.